plugin/src/operations.ts

```diff
@@ -135,45 +147,36 @@ export function createOperations(
       id: operationId,
     });
 
-    nodeStatsChangedCallback(operation.node);
-
-    while (true) {
-      if (options.verboseLogging) {
-        reporter.verbose(`
-          Waiting for operation to complete
-
-          ${operationId}
+    const completedTimer = reporter.activityTimer(
+      `Waiting for bulk operation to complete`
+    );
 
-          Status: ${operation.node.status}
-
-          Object count: ${operation.node.objectCount}
-
-          Url: ${operation.node.url}
-        `);
-      }
+    completedTimer.start();
 
+    while (true) {
       if (failedStatuses.includes(operation.node.status)) {
+        completedTimer.end();
```

Comment:
Anybody know what `activityTimer.panic` does? I find myself wondering if I should use that here.